### PR TITLE
DM-41702: Fix yaml formatting issue with the Scheduler.

### PR DESCRIPTION
**Pull request by tribeiro**

### CODE REVIEW

**Review by couger01 — APPROVED** [review]
LGTM.

File: python/lsst/ts/observing/block.py [hunk 1]
```diff
@@ -198,7 +198,18 @@ def get_script_configuration(self) -> str:
         config : `str`
             Script configuration.
         """
-        return yaml.safe_dump(self.parameters) if self.parameters else ""
```
Comment (timj) [inline python/lsst/ts/observing/block.py:201]: Is it possible that the problem is that parameters is defined as `dict[str, Any]` and that if it was restricted to the plain types we expect to be used that things would work without this change?

Comment (tribeiro) [reply python/lsst/ts/observing/block.py:201]: I was able to reproduce this issue standalone, just by getting a dictionary and doing the round trip, so I don't think so. 

Also I think the type annotation is ignored altogether by the runtime, no?

Comment (timj) [reply python/lsst/ts/observing/block.py:201]: The type annotation is critical for Pydantic. It coerces types and validates that they match. It's an important part of how pydantic works. If you know that the parameters are string or numbers and not arbitrary python types then we should probably say that regardless of this YAML bug.

I think you are saying that YAML writes out a sexagesimal string incorrectly? Can you put the actual error on the Jira ticket?

Comment (tribeiro) [reply python/lsst/ts/observing/block.py:201]: It is pretty simple to reproduce it. I will add it to the ticket.

```
In [1]: import sys

In [2]: import yaml

In [3]: a = dict(object_ra="+0:31:18")

In [4]: b = dict(object_ra="+1:31:18")

In [5]: yaml.safe_dump(a, sys.stdout)
object_ra: +0:31:18

In [6]: yaml.safe_dump(b, sys.stdout)
object_ra: '+1:31:18'
```

**Pull request merged**